Description: Check for libsystemd instead of libsystemd-daemon; libsystemd replaces libsystemd-daemon since systemd v209
Author: Michael Biebl <biebl@debian.org>
Orgin: vendor
Bug-Debian: #779770
Bug-Upstream: https://www.cups.org/str.php?L4669
Last-Update: 2015-07-13
--- a/config-scripts/cups-startup.m4
+++ b/config-scripts/cups-startup.m4
@@ -56,11 +56,11 @@
 	        	AC_MSG_ERROR(Need pkg-config to enable systemd support.)
                 fi
         else
-        	AC_MSG_CHECKING(for libsystemd-daemon)
-                if $PKGCONFIG --exists libsystemd-daemon; then
+        	AC_MSG_CHECKING(for libsystemd)
+                if $PKGCONFIG --exists libsystemd; then
                         AC_MSG_RESULT(yes)
-                        ONDEMANDFLAGS=`$PKGCONFIG --cflags libsystemd-daemon`
-                        ONDEMANDLIBS=`$PKGCONFIG --libs libsystemd-daemon`
+                        ONDEMANDFLAGS=`$PKGCONFIG --cflags libsystemd`
+                        ONDEMANDLIBS=`$PKGCONFIG --libs libsystemd`
                         AC_DEFINE(HAVE_SYSTEMD)
 			if test "x$SYSTEMD_DIR" = x; then
 			        SYSTEMD_DIR="`$PKGCONFIG --variable=systemdsystemunitdir systemd`"
